odl/test/solvers/functional/functional_test.py

```diff
@@ -449,6 +449,31 @@ def test_translation_of_functional(space):
                         places=places)
 
 
+def test_translation_proximal():
```

Comment:
There is already such a test here:

https://github.com/odlgroup/odl/blob/cfcd823c20b7747da8a4121fd2b5d588a0db285b/odl/test/solvers/functional/functional_test.py#L394

I feel like the naming of this should perhaps be more specific to distinguish them

Reply:
First, I tried to integrate the test there, which was not really elegant because of the space fixture. With the name I wanted to make clear that it is a test specifically for the proxmal of this operator. Would “test_translation_proximal_stepsizes” be clearer?

Reply:
Sounds mouch more clear to me!

Reply:
Okay, I changed the name (with git amend).